imgix/urlbuilder.py

```diff
@@ -82,7 +89,7 @@ def validate_domain(self, domain):
         if re.match(DOMAIN_PATTERN, domain) is None:
             raise ValueError(err_str)
 
-    def create_url(self, path, params=None):
+    def create_url(self, path="", params={}):
```

Comment:
qq: wasn't this function previously being exported? IE could be used in isolation by a user? 

ie: `builder.create_url("/bridge.png", {'w': 100, 'h': 100})`

If so changing the return value form an obj to a str would break their implementation. Idk if anyone in the wild is actually using this function, but worth noting.

Reply:
I don't follow where do you see that the return value has changed?